editor/styles/global.scss

```diff
@@ -328,7 +328,7 @@ to make sure it renders on top is a common fix */
 }
 
 .table-map-container {
-  height: 100%;
+  height: 85vh;
```

Comment:
Sorry for not providing a better spec for this layout challenge. In the interest of preserving the `100%` height behavior on the crashes and fatalities lists, I think this styling should do the trick:

* add a fixed height to the crashes `Card` component. `75vh` looks good to me?
* add `d-flex flex-column` to the the <Card.Body> component

<img width="1512" height="808" alt="Image" src="https://github.com/user-attachments/assets/f3124016-b944-47a2-8535-51cbad24b75d" />